R-packages/evalcast/R/calibration.R

```diff
@@ -35,14 +35,15 @@ compute_coverage <- function(score_card) {
       score_card %>%
         mutate(is_below = .data$actual < map_dbl(.data$forecast_distribution, ~ .x$quantiles[i]),
                is_above = .data$actual > map_dbl(.data$forecast_distribution, ~ .x$quantiles[itop]),
-               is_covered = !.data$is_below && !.data$is_above) %>%
+               is_covered = !.data$is_below & !.data$is_above) %>%
         group_by(.data$forecast_date) %>%
         summarize(prop_below = mean(.data$is_below, na.rm = TRUE),
                   prop_above = mean(.data$is_above, na.rm = TRUE),
                   prop_covered = mean(.data$is_covered, na.rm = TRUE))
     }, .id = "nominal_id") %>%
     mutate(nominal_coverage_prob = nominal_coverage_probs[as.integer(.data$nominal_id)]) %>%
```

Comment:
I believe we should have `.data$nominal_coverage_probs[as.integer(.data$nominal_id)]` here.

Reply:
Hmm that doesn't seem right because `nominal_coverage_probs` isn't a column of the data frame under consideration, it's an externally defined variable (see line 31).  So I left this as is.

Reply:
Oops, I was responding to a R CMD check NOTE, but I pointed to the wrong place.  Sorry.

Reply:
Ok, I've found the correct place and created a PR to this branch with you as reviewer in which I suggest the code change to address this.